src/common/options.h

```diff
@@ -39,17 +39,19 @@ struct Option {
    * Advanced: for users, configures some internal behaviour
    * Development: not for users.  May be dangerous, may not be documented.
    */
-  enum level_t {
-    LEVEL_BASIC,
-    LEVEL_ADVANCED,
-    LEVEL_DEV,
+  enum struct Level{
+    BASIC,
+    ADVANCED,
+    DEV,
   };
 
-  const char *level_to_str(level_t l) const {
-    switch(l) {
-      case LEVEL_BASIC: return "basic";
-      case LEVEL_ADVANCED: return "advanced";
-      case LEVEL_DEV: return "developer";
+  typedef Level level_t;
+
+  const char* level_to_str(level_t t) const {
```

Comment:
why do you think `t` is a better name?

Reply:
@tchaikov no idea :)

Reply:
@tchaikov that may probably be heritage of `Type`